orttraining/orttraining/core/graph/mixed_precision_transformer.cc

```diff
@@ -70,11 +70,13 @@ static const std::string loss_scale_input = "loss_scale";
 
```

Comment:
This file will be refactored anyway. So the purpose of this PR is to at least make current build, UTs, CI tests work and good to go?

Reply:
yes i need those change for internal EPs, i wish after this merge we can rebased the ort in internal repo to latest master, so we can get the latest update. without those changes, i can't run bfloat16 test internally.